pkg/grapicmd/internal/usecase/init_config.go

```diff
@@ -0,0 +1,34 @@
+package usecase
+
+import "bytes"
+
+type InitConfig struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type InitConfig should have comment or be unexported